README.md

```diff
@@ -1542,6 +1542,238 @@ data: {
 
 ```
 
+## Payment Channels
+
+Nakamoto style transaction replacement channels are supported in TXQ.
+
+https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2013-April/002417.html
+
+Use payment channels when you wish to coordinate actions amongst 2 or more parties, but do not wish to settle the transaction until
+nLockTime and the final state of the data is acceptable
+
+Note: Payment "Channels" have nothing directly related to the queue channel name used in the TXQ api. The name 'paymentChannelId' is used to distinguish the usage. In context below 'paymentChannelId' is a global identifier (preferred random uuid) that can be used by the client
+to create, update, and manage the Nakamoto replacement payment channel.
+
+### Create Payment Channel
+
+Choose a globally unique identifier for `paymentChannel` and create it. Optionally provide a `rawtx` for the initial transaction statte.
+
+Once a payment channel is initialized with the first transaction, then all subsequent updates must use the same inputs and adhere to the rules
+regarding nSequence, nLockTime. See: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2013-April/002417.html
+
+`POST /api/v1/paymentchannel/:paymentChannelId`
+
+```javascript
+{
+  "rawtx": '003222...' // Optional initial rawtx to initialize with. If left empty then use "Updates" to change
+}
+
+```
+
+### Add Transaction to Payment Channel
+
+`POST /api/v1/paymentchannel/:paymentChannelId/tx`
+
+```javascript
+{
+  "rawtx": '003222...' // Optional initial rawtx to initialize with. If left empty then use "Updates" to change
+}
+
+```
+
+Response:
+
+```javascript
+{
+  "status": 200,
+  "errors": [],
+  "result":  {
+    "closed": false,  // Whether the channel is closed manually or one of the inputs was spent by an actual tx broadcasted
+    "state": {
+      "id": 135,

```

Comment:
What exactly is `id` here? A random int?

Reply:
Only asking because we already have `paymentchannelid`

Reply:
`id` is the primary key of the tx (rawtx) in the database. The number is always guaranteed to be unique and increasing (but not necessarily consecutive, in other words it is monotonically increasing)

The purpose is an alternate `id` that you know is unique and chronologically ordered.  It let's you know if a state is newer than another one by mere virtue of the fact that the latest state has a higher `id`.

It's not necessarily to expose to the client, perhaps I can hide it here.  It may be useful to keep this in the `sse` updates because then it's easy to see when a state is newer.